corehq/apps/locations/util.py

```diff
@@ -47,7 +47,7 @@ def loc_to_json(loc, project):
         SQLLocation.root_locations(
             domain, include_archive_ancestors=include_archived
         )
-        if user_can_view_location(user, loc, project)
+        if user is None or user_can_view_location(user, loc, project)
```

Comment:
So having no user in the context is actually less restrictive than having a user configured to only view a subset of locations? 

That feels wrong/weird to me.